graviti/schema/base.py

```diff
@@ -47,37 +48,42 @@ def _repr1(self, level: int) -> str:
         with_params = False
         indent = level * _INDENT
         lines = [f"{self.__class__.__name__}("]
-        for name, parameter in self.params.items():
-            attr = getattr(self, name)
-            if attr != parameter.default:
```

Comment:
If we add a method `Param.check_default` to check wether the attr is equal to its default value, and handle the Fields logic in it?
Will it help?